### ISSUE TITLE
BUG - sev-3 - All - Some non-actionable boxes in the app flagged as interactive, with voice controls

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#9269]
## What happened?

Using voice control assistive technology, there are still a few remaining boxes in the app (even after changes in #8216) that are being flagged as interactive, even though you can't tap on them/they have no actions associated with them. Specifically, some text-only boxes such as the information on the prescription details screen, and the date of birth on the personal info screen:
<img width="350" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBemszQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--ef4ce3c2321b0d8540de4fe7b56a1c2952d8abef/image.png" alt="image.png" /> <img width="355" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBem8zQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--8e612fdec3208ccc6a1a4bf1c585aa9838e1022a/image.png" alt="image.png" />

## Specs:

- Device:
- OS:
- User Account:
- Accessibility State:

## Steps to Reproduce

Turn on voice control (typically in the device settings, in an accessibility submenu)
Speak the words "show numbers" (or, depending on platform, "show numbers continuously"). The intended functionality is for numbers to flag every element onscreen that is interactive (ie could be tapped if not using voice control)
Navigate to the listed screens in the app. Notice how read-only elements are flagged with numbers.

## Desired behavior

These elements should not be flagged as interactive. In 'show numbers' mode, they should not have numbers.

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: High Low 
- Frequency: High Low 

- 3 - Low
- 2 - High
- 1 - Critical

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))

**Comment by TKDickson** [comment]
Did a round of smoke testing in the app (I'm not going to attempt to look at every single instance of the list component), this is fixed in the specific instances I listed originally + I see it very few other places in the app.

Wrote a bug for the alert component for the design system team exhibiting the issue otherwise addressed in this ticket, [488 in their repo](https://github.com/department-of-veterans-affairs/va-mobile-library/issues/488)

I found an instance of this issue in the disability rating screen - I'm *not* asking for that to be fixed, given the problematic behaviors we've had with Android and disability rating with screen reader on (lots of duplicative read outs). 

I also found an instance of this on the direct deposit screen (see screenshot, the "Note: ..." content). Sending that back to @cadibemma for a fix. 
<img width="523" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNWpnQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--148acd5f966c3566f263fbc25b57792f8e5594f9/image.png" alt="image.png" />
